### DISCUSSION TITLE
API to list tags I am following?

### DISCUSSION THREAD

**Discussion by placoderm** [mastodon/mastodon#21284] [Q&A] [answered]
It was [mentioned here](https://github.com/mastodon/mastodon/issues/19895#issuecomment-1305104398) that there is an API for showing all the tags that a user has subscribed to.

Is there any documentation on this API and the APIs in general. I have some basic knowledge and would be interested in exploring the possibility of writing a script or plugin to add a list of tags being followed to my interface.

**Comment by trwnh** [comment, marked as answer]
https://docs.joinmastodon.org/methods/followed_tags/

**Reply by placoderm** [reply]
🤦🏻‍♂️ Sorry, I should have looked carefully at the documentation. 

I've never worked with OAuth before, so this will be a bit of a learning curve for me. Maybe someone else with more knowledge will be interested in this project too.

Thanks!

**Reply by trwnh** [reply]
There's a client developer guide/tutorial that might be helpful! https://docs.joinmastodon.org/client/intro/

**Reply by placoderm** [reply]
Thanks! 

I've worked with APIs in javascript, but never done anything that needed to be getting authentication. I'll see what I can figure out though. But hopefully this just gets added into Mastodon soon.
